frontend/src/components/CalcSpectrum.tsx

```diff
@@ -201,7 +201,7 @@ export const CalcSpectrum: React.FC = () => {
         link.href = url;
         link.setAttribute(
           "download",
-          `${data.mode}_${data.min_wavenumber_range}_${data.max_wavenumber_range}.spec`
+          `${data.database}_${data.species[0].molecule}_${data.min_wavenumber_range}_${data.max_wavenumber_range}cm-1_${data.tgas}K_${data.pressure}atm.spec`
```

Comment:
Should we do something like this in case the user has selected multiple molecules?
<img width="375" alt="Screen Shot 2022-08-22 at 10 38 11 AM" src="https://user-images.githubusercontent.com/13723264/185948361-97e2532b-6bb0-4a5b-8be5-5f5d54eb6a1b.png">